src/swap/events.ts

```diff
@@ -8,8 +8,10 @@ export enum SwapEventName {
   SWAP_ESTIMATE_GAS_CALL_FAILED = 'Swap Estimate Gas Call Failed',
   SWAP_FIRST_ACTION = 'Swap First Action',
   SWAP_FIRST_SIGNATURE_REQUESTED = 'Swap First Signature Requested',
-  SWAP_MAX_TOKEN_AMOUNT_SELECTED = 'Swap Max Token Amount Selected',
+  SWAP_MAX_TOKEN_AMOUNT_SELECTED = 'Swap Max Token Amount Selected', // TODO(xtine): remove after removing from universe codebase
```

Comment:
We can remove this comment - this whole package/file is going to be removed eventually, just tech debt!